Fortnox.NET/Models/InvoicePayment/CreateOrUpdateInvoicePayment.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FortnoxNET.Utils;
+
+namespace Fortnox.NET.Models.InvoicePayment
+{
+    [JsonPropertyClass("InvoicePayment")]
+    public class CreateOrUpdateInvoicePayment
```

Comment:
What do you think about introducing 'Response' and 'Request' as new terminologies? Seems like it could make naming things a bit easier instead of calling them XSubset or whatever which is not super clear and doesn't apply to every case.

Reply:
Sounds like a good idea, but let's handle that seperatley, please create an issue @BaconFinger